src/App.jsx

```diff
@@ -2,86 +2,54 @@ import React, { useState } from "react";
 import { ImSpades } from "react-icons/im";
 import { PiHeartStraightFill } from "react-icons/pi";
 import "./App.css";
+import { cardValues } from "./components/CardValues";
```

Comment:
Also, would be good if you move all constants to seperate constants.js file, for ex: src/helpers/constants.js

Reply:
created a folder "utils" and in it a file for constants